docs/self-hosted/docker-guide.md

```diff
@@ -7,47 +7,35 @@ readTime: 3 min read
 
 ::: info Non-Docker Guides
 
-We only publish and maintain self hosting guides using Docker as this removes many environment-specific configuration
+We only publish and maintain self-hosting guides using Docker as this removes many environment-specific configuration
 problems. If you can't or don't want to use Docker, we also publish an
 [npm package](https://www.npmjs.com/package/directus) without guides.
 
 :::
 
-Directus is published to [Docker Hub](https://hub.docker.com/r/directus/directus) under `directus/directus`. To use the
-latest Directus image from Docker Hub, run:
+Directus is published to [Docker Hub](https://hub.docker.com/r/directus/directus) under `directus/directus`. If you're just getting started, check out our [Self-Hosting Quickstart](/self-hosted/quickstart.html).
 
-```bash
-# Make sure to change sensitive values (KEY, SECRET, ...) in production
-docker run \
-  -p 8055:8055 \
-  -e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1 \
-  -e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 \
-  directus/directus
-```
-
-### Installing Specific Versions
+## Installing Specific Versions
 
 To stick to a more specific version of Directus you can use one of the following tags:
 
 - Full version, e.g. `10.0.0`
 - Minor releases, e.g. `10.0`
 - Major releases, e.g. `10`
 
-To use a specific version of Directus, run:
+To use a specific version of Directus, include the version number in your `services.directus.image` value:
 
-```bash
-# Make sure to change sensitive values (KEY, SECRET, ...) in production
-docker run \
-  -p 8055:8055 \
-  -e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1 \
-  -e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 \
-  directus/directus:10.0.0
+```yml
+services:
+  directus:
+    image: directus/directus // [!code --]
+    image: directus/directus:10.0.0 // [!code ++]
```

Comment:
Should we add a note that using an explicit version is the recommended way of doing things? It is generally not recommended to rely on the explicit or implicit `:latest` tag which doesnt work as most would assume https://www.howtogeek.com/devops/understanding-dockers-latest-tag#avoid-pinning-to-latest

Reply:
Yes, we should. Though in the quick start we mention latest, and I think for getting up and running that's the right suggestion. But I think as an effective follow-up guide, we should explicitly call this out.

Reply:
Yeah, could be as simple as changing the above sentence to
> To stick to a more specific version of Directus - which is highly recommended - you can use one of the following tags: